pyiron_workflow/snippets/files.py

```diff
@@ -75,6 +75,12 @@ def create_file(self, file_name):
     def is_empty(self) -> bool:
         return len(self) == 0
 
+    def remove(self, *files: str):
```

Comment:
If this is a function that deletes only files, then I would also call the function `remove_files`, because otherwise the distinction between `remove` and `delete` isn't so clear. If not, it at least merits a doc string that makes clear what's the difference.